R/generateEDGEdata.R

```diff
@@ -3,7 +3,8 @@
 #' Run this script to prepare the input data for EDGE in EDGE-friendly units and regional aggregation
 #'
 #' `calcgenerateEDGEdata()` is a wrapper for `generateEDGEdata()` to make use of
-#' madrat caching.
+#' madrat caching
+#' .
```

Comment:
Den hier müsstest du am besten noch rausnehmen :) Wenn du nichts bewusst an einem file geändert hast, würde ich das file bei github rauslassen beim commit

Reply:
soll ich einen file oder Zeilen 6 und 7 aus generateEDGEdata.R rauslöschen? 
#' madrat caching
#' .

Reply:
Einfach die Änderungen, die du ausversehen gemacht hast.. Glaube da ist nur ein Punkt in die nächste Zeile gerutscht oder so